snapcraft/extractors/_metadata.py

```diff
@@ -63,31 +63,35 @@ def get_summary(self) -> str:
         :returns: Extracted summary
         :rtype: str
         """
-        return str(self._data.get('summary'))
+        summary = self._data.get('summary')
```

Comment:
why not

```python
summary = self._data.get('summary', '')?

Reply:
I kinda like that the API returns `None` if the value wasn't extracted, but I don't think we have a use-case for that right now.

Reply:
fair enough